.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "sddf"]
 	path = examples/virtio/sddf
 	url = https://github.com/au-ts/sddf
-	branch = main
+	branch = main
```

Comment:
Remove this unrelated change.

Reply:
Just checked, this file is identical to `main` so I don't think I actually modified anything